assets/master.js

```diff
@@ -104,6 +105,19 @@
       byId('stats').innerHTML = '<span class="failure">' + escapeHTML(err) + '</span>';
       byId('minify-btn').disabled = false;
     });
+    byId('copy-btn').onclick = function() {
+      let bgColorButton = byId('copy-btn').style.backgroundColor;
+      let copyText = byId('output');
+      copyText.select();
+      copyText.setSelectionRange(0, 99999);
```

Comment:
This could lead to problems. Why not simply fetch the value and copy it with `window.Clipboard.writeText()`?

See https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText

Reply:
Sure, will make a commit doing that. What should be the response when the access is denied to write to the clipboard? Should we respond with `alert("Access denied to the clipboard")`?

Reply:
> Should we respond with `alert("Access denied to the clipboard")`?

Sounds good.

Reply:
Added this change in latest commit: b6249efc6e084ed7aa46ab511df9f811b36f4054